ffmpeg.h

```diff
@@ -25,6 +25,12 @@
 
 #endif
 
+/*
+ * AVPixelFormat is a newer release of avutil PixelFormat is the old name
+ */
+#ifndef AVPixelFormat
```

Comment:
Yup,  I took the idea out of kcompat usually found in kernel drivers. I'm not familiar with libavutil at all. So I'm sure there maybe a better way.